lua/barbar/bbye.lua

```diff
@@ -191,6 +191,12 @@ function bbye.delete(action, force, buffer, mods)
   -- Using buflisted() over bufexists() because bufhidden=delete causes the
   -- buffer to still _exist_ even though it won't be :bdelete-able.
   if buflisted(buffer_number) == 1 and buffer_number ~= get_current_buf() then
+    -- Disable auto commands when executing bdelete/bwipeout command
+    if type(mods) == 'string' then
+      mods = mods .. ' noautocmd'
+    elseif type(mods) == 'table' then
+      mods.noautocmd = true
+    end
```

Comment:
Why should autocommands be stopped in this case?

Reply:
I like this PR but I also question this modifier. We'll be preventing other plugins from doing their stuff.

Reply:
Because without this part of code, there's another error when delete a buffer using 'bdelete':

<img width="1920" alt="image" src="https://github.com/romgrk/barbar.nvim/assets/6496887/7bd0b705-7ee0-4adb-a0bc-cc78f2e00f63">

I will look into it, see why it's happening.

Reply:
Update:

After several trying, I still don't know why it's happening.

Reply:
And can you show the expanded stack trace, maybe with `:messages`?

Reply:
> Does that also happen with `nvim --clean`?

No, it doesn't.

Reply:
> And can you show the expanded stack trace, maybe with `:messages`?

After several days (I spent some days with my family), today I opened my macbook and as usual I just upgrade all my plugins.

This issue is just gone.... I didn't do anything.

Reply:
After several days back (I spent some days with family), I opened my macbook and upgrade all plugins and LSP servers (managed by mason.nvim) as usual.

And this issue is just gone..... I will notify if I had met this issue again.